From d059d0a4f352d295c116639798f8b5e110967e4f Mon Sep 17 00:00:00 2001
From: Fabian Winter <5821180+fdw@users.noreply.github.com>
Date: Mon, 27 Jun 2022 20:55:21 +0200
Subject: [PATCH] Return correct values from wofi

Returning no targets might not be the best idea...

Issue: #51
---
 src/rofi_rbw/selector.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/rofi_rbw/selector.py b/src/rofi_rbw/selector.py
index d2c9844..e5d8696 100644
--- a/src/rofi_rbw/selector.py
+++ b/src/rofi_rbw/selector.py
@@ -198,9 +198,9 @@ def show_selection(
             encoding='utf-8'
         )
         if wofi.returncode == 0:
-            return [], DEFAULT(), wofi.stdout
+            return DEFAULT(), DEFAULT(), wofi.stdout
         else:
-            return [], CANCEL(), wofi.stdout
+            return CANCEL(), CANCEL(), wofi.stdout
 
     def select_target(
         self,
